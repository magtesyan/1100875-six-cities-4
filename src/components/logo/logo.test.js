import renderer from "react-test-renderer";
import React from "react";
import {Router} from "react-router-dom";
import history from "../../history.js";
import Logo from "./logo.jsx";

const LOGO_TYPE = {
  HEADER: `header`,
  FOOTER: `footer`
};

it(`Logo is rendered correctly`, () => {
  const tree = renderer
    .create(
        <Router
          history={history}
        >
          <Logo
            onLogoClick={jest.fn()}
            logoType={LOGO_TYPE.HEADER}
          />
        </Router>
    ).toJSON();

  expect(tree).toMatchSnapshot();
});
