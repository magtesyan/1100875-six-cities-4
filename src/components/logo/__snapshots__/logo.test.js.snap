// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo is rendered correctly 1`] = `
<div
  className="header__left"
  onClick={[MockFunction]}
>
  <a
    className="header__logo-link"
    href="/"
    onClick={[Function]}
  >
    <img
      alt="6 cities logo"
      className="header__logo"
      height={41}
      src="img/logo.svg"
      width={81}
    />
  </a>
</div>
`;
